### ISSUE TITLE
Add comment 'from URL' and 'do not edit' to built-in devcontainer.json

### ISSUE THREAD

**Issue by chrmarti** [microsoft/vscode-dev-containers#392]
With https://github.com/microsoft/vscode-remote-release/issues/3293 it has become very easy for new users to open a built-in devcontainer.json. For their info and to prevent them loosing changes we should include a comment at the top of every built-in config pointing the user to the source in this repository and advising them not to edit the file because the changes will be lost when the extension updates.

**Comment by Chuxel** [comment]
@chrmarti Just to clarify, are you talking about for `repository-containers` specifically?  If so, that's a good call out. (For `containers`, there's already a pointer.)

**Comment by chrmarti** [comment]
Yes, those configurations in `repository-containers`. We don't have that many yet, but with the (delayed) automatic activation of Remote-Containers and the link for opening the devcontainer.json in the Reopen notification, it seems likely that we will see (especially new) users edit these built-in configurations and then loosing their changes when the extension updates.

Not sure how many of these we will see, but maybe we can at least include something minimal at the top of each for this milestone. Like:
```jsonp
// NOTE: Changes to this file and files in this folder will be lost on the next update of the Remote-Containers extension.
```

References to the documentation and the source of the config would make sense too.

**Comment by Chuxel** [comment]
@chrmarti Yeah I was thinking we could actually tell people that if they want to make modifications they can copy the contents to a local repository folder now that we have that.

**Comment by chrmarti** [comment]
I agree, we can also think about adding a command to help with that.

**Comment by chrmarti** [comment]
We could offer to copy a configuration to one of the configuration folders in the settings if there is one and also to the workspace folder through a command. I'd first wait and see if this is indeed something users request / need, but there are definitely options for improvement.

**Comment by chrmarti** [comment]
Current suggestion for the June milestone is to open the devcontainer.json in the browser on GitHub: https://github.com/microsoft/vscode-remote-release/issues/3322. We that we don't need the comments for now.
